config/searchguard/sg_roles_mapping.yml

```diff
@@ -14,6 +14,14 @@ kibana_user:
     users:
         - kibana
 
+read_only_user:
```

Comment:
what is the difference between read_only_user and reader_user?
You renamed the read_only role to reader_user

Reply:
it was not used at all previously , so its like i created it